db/migrate/20210819170858_create_screenings.rb

```diff
@@ -0,0 +1,11 @@
+# frozen_string_literal: true
+
+class CreateScreenings < ActiveRecord::Migration[6.1]
+  def change
+    create_table :screenings do |t|
+      t.datetime :date_and_time, null: false
+
```

Comment:
Thanks, I remember now: I planned to add FKs after all my models will be implemented and this will happen after adding devise and creating User model, but I agree - there's no need to postpone adding Foreign Keys for already existing associations.

Therefore: Foreign Keys are added.